src/Microsoft.ML.Data/Model/Onnx/OnnxContext.cs

```diff
@@ -23,6 +23,13 @@ public abstract class OnnxContext
         /// <returns>A name that has not yet been returned from this function, starting with <paramref name="prefix"/></returns>
         public abstract string GetNodeName(string prefix);
 
+        /// <summary>
+        /// Determine if a string has been used as ONNX variable name somewhere.
+        /// </summary>
+        /// <param name="variableName">examined string</param>
+        /// <returns>True if the input argument has been used to denote an ONNX variable. Otherwise, False.</returns>
+        public abstract bool IsDefined(string variableName);
```

Comment:
>IsDefined [](start = 29, length = 9)

Can we please rename this to be more specific like "IsVariableDefined"?

Reply:
Sure.

---
In reply to: [222023194](https://github.com/dotnet/machinelearning/pull/952#discussion_r222023194) [](ancestors = 222023194)